docs/06-contributors/99-maintainers/050-operations.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Operations Overview
+id: operations
+keywords: [operations]
+---
+
+## Coverage
+
+"Wing" encompases several projects/sites.
+This document covers operations for the following:
+
+- https://docs.winglang.io (https://github.com/winglang/docsite)
+- Wing (https://github.com/winglang/wing)
+- Wing Console (https://github.com/winglang/console)
```

Comment:
Let's add:
- the upcoming winglang/examples repo here (paging @skorfmann)
- Also the winglang.io landing page
- playground
- monitor published packages in npm, console bucket, vscode extension